met-web/src/routes/AuthenticatedRoutes.tsx

```diff
@@ -88,33 +93,58 @@ const AuthenticatedRoutes = () => {
                     id="single-engagement"
                     errorElement={<NotFound />}
                     loader={engagementLoader}
-                    handle={{
-                        crumb: async (data: { engagement: Promise<Engagement> }) => {
-                            return data.engagement.then((engagement) => {
-                                return {
-                                    link: `/engagements/${engagement.id}/view`,
-                                    name: engagement.name,
-                                };
-                            });
-                        },
-                    }}
                 >
-                    <Route index element={<Navigate to="view" />} />
                     <Route element={<AuthGate allowedRoles={[USER_ROLES.EDIT_ENGAGEMENT]} />}>
                         <Route path="form" element={<EngagementForm />} />
-                        <Route
-                            path="config"
-                            element={<EngagementConfigurationWizard />}
-                            action={engagementUpdateAction}
-                            handle={{
-                                crumb: () => ({
-                                    name: 'Configure',
-                                }),
-                            }}
-                        />
                     </Route>
                     <Route path="old-view" element={<OldEngagementView />} />
-                    <Route path="view" element={<AdminEngagementView />} />
+                    <Route
+                        path="view"
+                        loader={engagementLoader}
+                        handle={{
+                            crumb: async (data: { engagement: Promise<Engagement> }) => {
+                                return data.engagement.then((engagement) => {
+                                    return {
+                                        link: `/engagements/${engagement.id}/details/config`,
+                                        name: engagement.name,
+                                    };
+                                });
+                            },
+                        }}
+                        element={<AdminEngagementView />}
+                    />
+                    <Route index element={<Navigate to="details/config" />} />
+                    <Route path="details">
+                        <Route index element={<Navigate to="config" />} />
+                        {/* Wraps the tabs with the engagement title and TabContext */}
+                        <Route element={<AdminEngagementView />}>
+                            <Route path="config" element={<ConfigTab />} />
+                            <Route path="authoring" element={<AuthoringTab />}></Route>
```

Comment:
Add a splat route up here to provide the navigation tabs for lost users
```suggestion
                            <Route path="authoring" element={<AuthoringTab />}></Route>
                            <Route path="*" element={<NotFound />} />
```